pagopa/io-app
# feat(IT Wallet): [SIW-1022] Add PIN/biometric authorization for eID store action

> [!WARNING]
> This PR depends on https://github.com/pagopa/io-app/pull/5780


## Short description
This PR adds the PIN/biometric authorization request before proceeding to the eID store.

## List of changes proposed in this pull request
- Added `useItwDismissalDialog` hook which shows a confirmation dialog for the dismissal of the current procedure
- Added `ItwIssuanceEidResultScreen` screen and routes
- Added required locales
- Within the `ItwIssuanceEidPreviewScreen` screen, it now show the PIN/biometric auth request when tapping the "Store" button.

## How to test
Navigate to **Profile > Playgrounds > IT Wallet > eID > SPID**, select an IDP. In the eID preview screen, tap on "Store to wallet" button. You should be able to see the PIN/biometric authorization request.

## Preview

https://github.com/pagopa/io-app/assets/6160324/5ef9a8a6-400a-4c17-8428-10fd2b28e37b
